barretenberg_wasm/src/composer.rs

```diff
@@ -346,7 +362,7 @@ mod test {
         // This fails because the constraint system requires public inputs,
         // but none are supplied in public_inputs. So the verifier will not
         // supply anything.
-        let case_1 = WitnessResult {
+        let _case_1 = WitnessResult {
```

Comment:
can we remove the prepending underscore?

Reply:
Not without removing the tests. These are broken at barretenberg, which panics.